<ui version="4.0" >
 <class>NetGraphConfig</class>
 <widget class="QDialog" name="NetGraphConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>Device :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="deviceLineEdit" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Limits :</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="1" column="1" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>KBit/s</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Download Speed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QDoubleSpinBox" name="dSpinBox" >
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>128.000000000000000</double>
          </property>
          <property name="value" >
           <double>1024.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="1" column="1" >
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>KBit/s</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>Upload Speed</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QDoubleSpinBox" name="uSpinBox" >
          <property name="decimals" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <double>1000000000.000000000000000</double>
          </property>
          <property name="singleStep" >
           <double>128.000000000000000</double>
          </property>
          <property name="value" >
           <double>1024.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>Show Grid : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="gridCheckBox" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Orientation :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="orientationComboBox" >
        <property name="maxVisibleItems" >
         <number>2</number>
        </property>
        <property name="maxCount" >
         <number>2</number>
        </property>
        <item>
         <property name="text" >
          <string>Horizontal</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Vertical</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_7" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_14" >
        <property name="text" >
         <string>Graph Type :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="graphTypeComboBox" >
        <item>
         <property name="text" >
          <string>Peaks</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Bars</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Line</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_4" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_9" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Colors :</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Download Graph</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="dColorButton" >
          <property name="color" >
           <color>
            <red>0</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </property>
          <property name="defaultColor" >
           <color>
            <red>0</red>
            <green>255</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Upload Graph</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="uColorButton" >
          <property name="color" >
           <color>
            <red>255</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </property>
          <property name="defaultColor" >
           <color>
            <red>255</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="blendCheckBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>&lt;html>&lt;body>&lt;div>Blend&lt;br>into&lt;br>background.&lt;/div>&lt;/html>&lt;/body></string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_6" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_13" >
        <property name="text" >
         <string>Graph Detail:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="detailSlider" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="value" >
         <number>10</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="label_12" >
        <property name="text" >
         <string>Engine Update Interval (msec):</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QSpinBox" name="intervalSpinBox" >
        <property name="minimum" >
         <number>500</number>
        </property>
        <property name="maximum" >
         <number>999999999</number>
        </property>
        <property name="singleStep" >
         <number>500</number>
        </property>
        <property name="value" >
         <number>1000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetGraphConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetGraphConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
